packages/snaps-simulation/src/helpers.ts

```diff
@@ -126,6 +126,20 @@ export type SnapHelpers = {
     keyringRequest: KeyringOptions,
   ): Promise<SnapResponseWithoutInterface>;
 
+  /**
+   * Get the response from the Snap's `onInstall` handler.
+   *
+   * @returns The response.
+   */
+  onInstall(request?: RequestOptions): SnapRequest;
```

Comment:
```suggestion
  onInstall(request?: Pick<RequestOptions, 'origin'>): SnapRequest;
```

I think we can simply do this for now

Reply:
Updated.